setup.py

```diff
@@ -24,45 +23,37 @@ def read(*filenames, **kwargs):
 
 
 root = os.path.dirname(os.path.realpath(__file__))
-version_module = imp.load_source(
-    "version", os.path.join(root, "{{ PACKAGE }}", "version.py"))
-testing = "test" in sys.argv or "pytest" in sys.argv
+version = runpy.run_path(os.path.join(
+    root, 'nengo_bones', 'version.py'))['version']
 
-install_requires = [
-]
-docs_require = [
-    "sphinx",
-    "nengo_sphinx_theme",
-]
-optional_requires = [
-]
-tests_require = [
-    "pytest>=3.6",
-]
+install_requires = ["click>=7.0", "jinja2>=2.10", "pyyaml>=5.1"]
 
+# note: these are intentionally empty to test that the correct requirements
+# are built into the ci scripts
+tests_require = []
+docs_require = []
 
 setup(
-    name="{{ REPO NAME }}",
-    version=version_module.version,
+    name="nengo-bones",
+    version=version,
     author="Applied Brain Research",
     author_email="info@appliedbrainresearch.com",
     packages=find_packages(),
-    scripts=[],
-    data_files=[],
-    url="https://github.com/nengo/{{ REPO NAME }}",
+    url="https://github.com/nengo/nengo-bones",
     license="Free for non-commercial use",
-    description="{{ DESCRIPTION }}",
+    description="Tools for managing Nengo projects",
     long_description=read("README.rst", "CHANGES.rst"),
-    setup_requires=["pytest-runner"] if testing else [] + install_requires,
     install_requires=install_requires,
     extras_require={
-        "all": docs_require + optional_requires + tests_require,
+        "all": docs_require + tests_require,
         "docs": docs_require,
-        "optional": optional_requires,
         "tests": tests_require,
     },
-    tests_require=tests_require,
-    classifiers=[
-        # See https://pypi.python.org/pypi?%3Aaction=list_classifiers
-    ],
+    classifiers=[],  # TODO
```

Comment:
Just noticed... ^ should this be done in this PR?

Reply:
I was just going to wait until the next step, when this file will be templated anyway.

Reply:
Okay. The main consideration was in using the `Framework :: Nengo` classifier for the release, as the PyPi team requested "immediate use to existing projects". But if the next release after is happening soon then that's all right.